src/main/webapp/util/TableUtil.js

```diff
@@ -0,0 +1,57 @@
+import { isEmpty } from "./Utils";
+
+export const TABLE_ACTIONS = {
+  SEARCH : "search",
+  FILTER: "filter",
+  PAGINATION: "pagination",
+  SORT: "sort"
+};
+
+export const formatNullCell = (cell)  => {
+  return cell === null ? '' : cell
+};
+
+// method used to transform data format into a single array to be used for table printing
+export const formatData = (data, columns) => {
```

Comment:
nit: can this be named more appropriately to what it's doing, i.e. `formatDataPrintableFormat` or something like that?

Reply:
Yes! i changed its name.